script/Arcueid/Day 11/11_00_ARC11_3B.txt

```diff
@@ -428,17 +442,17 @@ Damn, I'm breathing too heavily.
 [sha:0ce5f4cd49f97505d09136abddddf6bffdda7279]{
 -- Page 280, Offset 30331.
 -- でも同じ過ちは繰り返せない。
-I won't repeat the same mistake.
+I won't make the same mistake a second time.
 }
 [sha:8f13df8a79e04777743f651f4ad06f2a5b1fc26c]{
 -- Page 280, Offset 30332.
 -- どんなに彼女が欲しくても、もう自分だけの気持ちはぶつけられない。
-No matter how much I want her, I can't just let loose on her.
+No matter how much I want her, I can't just give in to my own emotions at the expense of hers.
 }
 [sha:b3b9ea79d3e348410c1e8a94ab919326c39a60d1]{
 -- Page 281, Offset 30333.
 -- ……アルクェイドはそんな俺の顔を見上げて、
-Arcueid is looking up at the face I'm making.
+...Arcueid looks up and sees the face I'm making.
 }
 [sha:eaf378b9049d2cf8675069f84dbd7ea00467e580]{
 -- Page 281, Offset 30334.
```

Comment:
I wanted to specifically bring up this line, partially because its by far the worst line in the game, and partially because it's kinda wrong.
> ……うん。いいよ、志貴。あの時の続き、しよ？

あの時 is totally missing and what she's actually saying is should we continue from _back in the alley_. Like, during the _sexual assault_ scene. Got any ideas on how to work that in? Because right now it just sounds like she's saying "lets go all the way" instead of "lets continue from when you were brutally assaulting me in the alley"

Reply:
Yeah, I noticed that but assumed it was intentionally dropped and was kinda ok with it. I suppose that we should favour correctness though. 
> Shall we... pick up where we left off?

?

Reply:
I think that's good if you use "pick up from before". Maybe italicize before.